easybuild/tools/config.py

```diff
@@ -193,7 +193,7 @@ def mk_full_default_path(name, prefix=DEFAULT_PREFIX):
         'zip_logs',
     ],
     False: [
-        'add_dummy_to_minimal_toolchains',
+        'add_system_to_minimal_toolchains',
```

Comment:
This one will be important to mention in the release notes!

Reply:
Yep, which is why I also mentioned it in the PR description (of course this is only for EasyBuild v4.0)